domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleResponse.java

```diff
@@ -107,6 +107,19 @@ public PartAdmissionResponse buildWithPaymentOptionInstallments() {
                 .statementOfMeans(SampleStatementOfMeans.builder().build())
                 .build();
         }
+
+        public PartAdmissionResponse buildWithPaymentOptionInstallmentsAndParty(Party party) {
```

Comment:
only 1 L in installments :)

Reply:
Extra L is the superstitious 'L' 😆. Rectifying it!